sv-gpl/ktx/port_template.cfg

```diff
@@ -2,7 +2,7 @@
 //set k_motd1                   "" // auto generated server name at the bottom of the config
 set	k_motd2                     " "
 set	k_motd3                     "Available game modes:"
-set	k_motd4                     "1on1, 2on2, 4on4, 10on10, ffa, ctf"
+set	k_motd4                     "1on1, 2on2, 4on4, 10on10, XonX, ffa, ctf,"
```

Comment:
Should there be a trailing comma?

Reply:
@nvtkaszpir Is that trailing comma after ctf an issue? Please remove it :)

I'm a bit swamped at work at the moment but can always press the button if there are no uncertainties.

Reply:
The comma is there because k_motd5 is continuation of the context, that's why I added it.
If you feel this is not needed then ok.

Reply:
Thanks for clarifying, I should read more carefully next time. :)